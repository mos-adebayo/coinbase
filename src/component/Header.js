import React from 'react';
import logo from '../logo.svg';

const Header = () => (
    <div className="navbar-fixed">
        <nav className={'blue-grey darken-4'}>
            <div className="nav-wrapper container">
                <a href={'/'} className="brand-logo">
                    <img src={logo} width={55} alt="Donut"/>
                </a>
            </div>
        </nav>
    </div>
);

export default Header;
