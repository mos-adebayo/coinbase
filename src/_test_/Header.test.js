import React from "react";
import ReactDOM from "react-dom";
// import { configure, shallow } from "enzyme";
import renderer from 'react-test-renderer';

import Header from "../component/Header";

//Enzyme Configurations
// import Adapter from 'enzyme-adapter-react-16';
// configure({ adapter: new Adapter() });

test('Header components snapshots test', () => {
    const component = renderer.create(
        <Header />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
    ReactDOM.unmountComponentAtNode(div);
});
