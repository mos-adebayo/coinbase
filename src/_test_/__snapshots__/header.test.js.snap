// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header components snapshots test 1`] = `
<div
  className="navbar-fixed"
>
  <nav
    className="blue-grey darken-4"
  >
    <div
      className="nav-wrapper container"
    >
      <a
        className="brand-logo"
        href="/"
      >
        <img
          alt="Donut"
          src="logo.svg"
          width={55}
        />
      </a>
    </div>
  </nav>
</div>
`;
